cmd/neofs-node/policy.go

```diff
@@ -154,24 +154,24 @@ func (x *containerNodes) forEachContainerNode(cnrID cid.ID, withPrevEpoch bool,
 // the underlying storage, applies the storage policy to it and returns sorted
 // lists of selected storage nodes along with the per-list numbers of primary
 // object holders. Resulting slices must not be changed.
-func (x *containerNodes) getNodesForObject(addr oid.Address) ([][]netmapsdk.NodeInfo, []uint, []iec.Rule, error) {
+func (x *containerNodes) getNodesForObject(addr oid.Address) storagePolicyRes {
```

Comment:
I'd expect `struct{...}, error` at least. Or just leaving as is given that public `GetNodesForObject()` is still the same.